screeps-game-api/src/lib.rs

```diff
@@ -11,13 +11,71 @@ extern crate serde_derive;
 #[macro_use]
 extern crate stdweb;
 
+/*  Used to get data from a javascript reference back into rust code. 
```

Comment:
Style wise, I'd prefer doc comments with `//!` rather than than `/*`. Not a big deal, I can definitely change this after merging if you're alright with that.

The practical advantages of doc comments include code examples being tested with `cargo test` and inclusion in editor tooltips when using `rls` or other rust-IDE integration.

Reply:
Darn it, meant `///` here instead of `//!`. Not sure what I was thinking.